src/main/charts/bitbucket/values.yaml

```diff
@@ -1553,3 +1553,59 @@ openshift:
   # configuration files as ConfigMaps.
   #
   runWithRestrictedSCC: false
+
+# OpenSearch config
+#
+opensearch:
+
+  # -- Deploy OpenSearch Helm chart and automatically
+  # configure Bitbucket to use it as a search platform
+  #
+  enabled: false
+
+  # -- The base URL of the external OpenSearch instance to be used, for example:
+  # http://opensearch-master.<namespace>.svc.cluster.local:9200
+  # If this is defined and bitbucket.opensearch.enabled is set to false,
```

Comment:
This is slightly confusing. I can have:
`opensearch.enabled: false`
`opensearch.baseUrl: https://aws.opensearch.url` and it means that opensearch will be enabled on Bitbucket but it won't be installed. Would be more accurate to call it:
`opensearch.install: true` and separate it from `opensearch.enabled`?